_posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md

```diff
@@ -53,11 +53,15 @@ Please read the articles below to prepare for next week's class.
 
 
 
-
-
-
-
-
+{:.output}
```

Comment:
@nkorinek can you hide the output of this download so it doesn't print in the lesson? also is this a PR that you want merged? it says "testing"